rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/20_synonyms_get.yml

```diff
@@ -0,0 +1,129 @@
+setup:
+  - do:
+      synonyms.put:
+        synonyms_set: test-get-synonyms
+        body:
+          synonyms_set:
+            - synonyms: "hello, hi"
+              id: "test-id-1"
+            - synonyms: "bye => goodbye"
+              id: "test-id-2"
+            - synonyms: "test => check"
+              id: "test-id-3"
+
+---
+"Get synonyms set":
+  - skip:
```

Comment:
We can the part below in the `setup` section, so we don't need to repeat it for each test section

```
  - skip:
      version: " - 8.8.99"
      reason: Introduced in 8.9.0
```

Reply:
Nice!